libultraship/libultraship/Lib/Fast3D/gfx_opengl.cpp

```diff
@@ -31,10 +31,15 @@
 #elif __APPLE__
 #include <SDL.h>
 #include <GL/glew.h>
+#elif __SWITCH__
+#include <SDL2/SDL.h>
+#include "glad/glad.h"
```

Comment:
we're importing a lib here
```suggestion
#include <glad/glad.h>
```